lib/config.ts

```diff
@@ -58,6 +60,8 @@ export const getRataExtraStackConfig = (scope: Construct) => ({
   dmzApiEndpoint: getSSMStringParameter(scope, SSM_DMZ_API_DOMAIN_NAME),
   databaseDomain: getSSMStringParameter(scope, SSM_DATABASE_DOMAIN),
   jwtTokenIssuer: getSSMStringParameter(scope, SSM_JWT_TOKEN_ISSUER),
+  alfrescoAPIKeyName: getSSMStringParameter(scope, SSM_ALFRESCO_API_KEY_NAME),
```

Comment:
This will try to get the value. Unsuccessfully, as this is a SecuredString. Instead, the current solution is to pass the e.g. `SSM_ALFRESCO_API_KEY_NAME` directly as a string to the specified stack (rataextra-backend.ts in this case). I'm not happy with the current situation and might refactor it, but that's the current situation.